sql/synth_recipes.sql

```diff
@@ -60,55 +60,6 @@ CREATE TABLE `synth_recipes` (
 -- Dumping data for table `synth_recipes`
 --
 
-DELIMITER $$
```

Comment:
Revert it to how it was before without the `-`. You can just copy and paste it back. You'll then need to reimport the table to make sure it doesn't fail the checks.

Ingredients have to be ordered from lowest number to highest or else they won't be recognized.